torch/autograd/function.py

```diff
@@ -319,9 +319,9 @@ class _SingleLevelFunction(
 ):
     @staticmethod
     def forward(ctx: Any, *args: Any, **kwargs: Any) -> Any:
-        r"""
-        This function is to be overridden by all subclasses. There are two ways
-        to define forward:
+        r"""Override this function in all subclasses.
```

Comment:
I think the old wording is preferred here.

Reply:
Old wording is giving imperative mood error. So, I have now changed it to "Use this function and override it in all subclasses.", Please review this. Thank you.